macos/FluentUI/Link.swift

```diff
@@ -110,7 +121,10 @@ open class Link: NSButton {
 	}
 	
 	private func updateTitle() {
-		let titleAttributes = (showsUnderlineWhileMouseInside && mouseEntered) ? underlinedlinkAttributes: linkAttributes
+		
+		let shouldUnderline = showsUnderlineWhileMouseInside && mouseEntered
```

Comment:
Why split this out?

Reply:
It just felt slightly cleaner to me, no strong reason.